cloudify_marketplace/resources/replace_ssl_certificates

```diff
@@ -146,6 +148,19 @@ if __name__ == '__main__':
             AMQPINFLUX_CONFIG_PATH,
         ])
 
+        # Update the manager credentials
+        username = details['new_manager_username']
```

Comment:
Can we differentiate this as manager_username (or even new_manager_username) to avoid any confusion with the broker credentials? (same with password)